client/src/components/Topbar/Topbar.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import './Topbar.css';
+
+function Topbar() {
+    return <div className='topbar'>
```

Comment:
It is good practice to wrap the return content from a function in parentheses when returning multi-line jsx.